### ISSUE TITLE
PogoPlug mobile no longer allowing SSH

### ISSUE THREAD

**Issue by radostyle** [pepaslabs/pogoplug_mobile_uboot_installer#2]
They've closed the https backdoor to enable SSH.  I bought a PogoPlug Mobile and the backdoor did not work.

http://forum.doozan.com/read.php?2,28710

**Comment by cellularmitosis** [comment]
Oh man, that is a bummer :(

**Comment by cellularmitosis** [comment]
@radostyle How did you try to enable SSH?  Did you try the "curl" method?

e.g.

`curl -k "https://root:ceadmin@192.168.X.X/sqdiag/HBPlug?action=command&command=dropbear%20start"`

**Comment by radostyle** [comment]
Yes, I used the curl method to enable SSH, but HTTPS was not available so
the curl request was denied.  Nmap also showed only port 80 and 3333 being
available.  So I ended up having to solder wires to the serial bus inside
to get  access.

**Comment by radostyle** [comment]
Here are the instructions I used to wire it for serial communication:
http://mhassan.me/2012/11/17/pogoplug-v4-disassembly-and-attaching-serial-cable/

Here is the USB serial converter I used:
https://www.amazon.com/gp/product/B009T2ZR6W

Then I used screen to communicate as documented here:
http://varkey.in/seagate-goflex-net-serial-connection/

**Comment by november17** [comment]
I can confirm that. Also Pepa is your hosting working correctly? I was getting an error when I tried to pull files off your domain. Thanks

**Comment by cellularmitosis** [comment]
Thanks guys, I just dug one of these out of the closet and can confirm I had to solder a serial connection to it in order to root it.
